res/values/strings.xml

```diff
@@ -505,9 +503,9 @@
     <string name="systemmsg_member_added">Member %1$s added.</string>
     <string name="systemmsg_member_removed">Member %1$s removed.</string>
     <string name="systemmsg_group_left">Group left.</string>
-    <string name="systemmsg_read_receipt_subject">Read receipt</string>
-    <string name="systemmsg_read_receipt_body">This is a read receipt for the message \"%1$s\".\n\nIt means that message was displayed on the recipient\'s device, not necessarily that the content was read.</string>
-    <string name="systemmsg_cannot_decrypt">This message cannot be decrypted.\n\n• It might already help to simply reply to this message and ask the sender to send the message again.\n\n• In case you re-installed Delta Chat or another email program on this or another device you may want to send an Autocrypt Setup Message from there.</string>
+    <string name="systemmsg_read_receipt_subject">Message opened</string>
+    <string name="systemmsg_read_receipt_body">The \"%1$s\" message you sent was displayed on the screen of the recipient.\n\nThis is no guarantee the content was read.</string>
+    <string name="systemmsg_cannot_decrypt">This message cannot be decrypted.\n\n• Reply to this message and ask the sender to try sending it again.\n\n• In case you have re-installed Delta Chat or another email app on this or another device, send it an Autocrypt Setup Message.</string>
```

Comment:
The string now suggests that the solution for the failing decryption is a reply. But there might be other reasons why the decryption can fail. That's why we're in favor of a more vague statement.

Reply:
I reverted this. Is it good?